### DISCUSSION TITLE
Please add cartoon mode to PDF as well.

### DISCUSSION THREAD

**Discussion by mark10als** [sumatrapdfreader/sumatrapdf#3298] [Q&A] [answered]
In Japan, some people publish their manga in PDF format.
Therefore, I want you to be able to display it in "manga mode" even when you open the PDF.

**Comment by GitHubRulesOK** [comment, marked as answer]
You can use manga mode in PDF (and set RTL OR LTR by view on opening preference) , for swap left/right pages (all it is intended to do) but need to assign to a key of your choice, if you want to flip over while viewing..
![image](https://user-images.githubusercontent.com/12754913/221054743-0e6eaed7-78c1-4f42-a00d-8e4c55bb1002.png)


HOWEVER it will not remove the voids between images only the voids between pages, (this was raised in the past) 
thus images should touch paper media edges, without void white space.
![image](https://user-images.githubusercontent.com/12754913/221056853-5e24b1d7-59ec-41ed-97fc-093d90bd1864.png)

**Comment by mark10als** [comment]
Thank you for your answer.
Solved by adding a shortcut key to the setting file "SumatraPDF-settings.txt".
In American comics and manga files (CBZ), there was a "manga mode" in the "view" of the "menu", so I thought that it would be the same for PDF files.
This query will be closed.
